src/tool/subcommands/api_cmd.rs

```diff
@@ -671,6 +717,237 @@ async fn compare_apis(
     run_tests(tests, &forest, &lotus, &config).await
 }
 
+async fn start_server(
+    snapshot_path_opt: Option<PathBuf>,
+    chain: NetworkChain,
+    rpc_port: u16,
+) -> anyhow::Result<()> {
+    println!("Configuring Node-Less RPC Server");
+    let db_path = tempfile::Builder::new().tempdir()?.path().join("car-db");
+    let db = Arc::new(ParityDb::open(&db_path, &ParityDbConfig::default())?);
+
+    let (snapshot_file, snapshot_path) = if let Some(path) = snapshot_path_opt {
+        (File::open(&path).await?, path)
+    } else {
+        println!(
+            "No snapshot provided, downloading latest snapshot for {}",
+            chain
+        );
+        let snapshot_url =
+            crate::cli_shared::snapshot::stable_url(TrustedVendor::default(), &chain)?;
+        let tmp_snapshot_path = tempfile::Builder::new().tempdir()?.into_path();
```

Comment:
`/tmp` typically has 10GB size limit which is insufficient for downloading a mainnet snapshot

Reply:
I think we could just keep the logic simple, ask users to download a snapshot via `forest-tool snapshot fetch`

Reply:
Or use logic similar to that of the running node?

Reply:
I agree with roman, I think we should keep the automatic snapshot download feature if a snapshot is not provided

Reply:
Yeah, but you have to take into account Hailong's comment about `tmp` dir size.

Reply:
using project dir now